app/containers/MessageBox/index.js

```diff
@@ -932,7 +932,7 @@ class MessageBox extends Component {
 					keyboardType='twitter'
 					blurOnSubmit={false}
 					placeholder={I18n.t('New_Message')}
-					placeholderTextColor={themes[theme].auxiliaryTintColor}
+					placeholderTextColor={themes[theme].bodyText}
```

Comment:
This line was correct.
You just have to add this theme [here](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/containers/MessageBox/styles.js#L19) to `color`.

Try adding `color: 'red',` there and you'll see.

Reply:
So you are saying PlaceHolderText would be auxiliaryTintColor and Color of text message should be bodyText?